<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QDialog" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>667</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>TubeTK Image Viewer</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1021</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QToolButton" name="ZoomIn">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
             </font>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ZoomOut">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
             </font>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="horizontalSpacing">
           <number>2</number>
          </property>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_Right">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>R</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_Left">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>L</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_Center">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_Up">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>U</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_Down">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>D</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="Display2D">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Value</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inverse</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Log</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Deriv X</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Deriv Y</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Deriv Z</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blend</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MIP</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="PositionX">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="PositionY">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="PositionZ">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="PixelValue">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QtGlSliceView" name="Window2D" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="Orientation2D">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>50</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>XY</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>XZ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>YZ</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="SliceNum">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>50</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSlider" name="Slice">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="invertedAppearance">
             <bool>true</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
            <property name="tickInterval">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QtGlSliceView" name="Window3D" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="XY3D">
          <property name="text">
           <string>XY</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="XZ3D">
          <property name="text">
           <string>XZ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="YZ3D">
          <property name="text">
           <string>YZ</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
      <item>
       <widget class="QLabel" name="IntensityMinLabel">
        <property name="text">
         <string>Int. Min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="IntensityMinDisplay"/>
      </item>
      <item>
       <widget class="QSlider" name="IntensityMin">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="IntensityMaxLabel">
        <property name="text">
         <string>Int. Max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="IntensityMaxDisplay"/>
      </item>
      <item>
       <widget class="QSlider" name="IntensityMax">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="HelpButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="text">
         <string>&amp;Help</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOk">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="windowTitle">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Quit</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtGlSliceView</class>
   <extends>QWidget</extends>
   <header>QtGlSliceView.h</header>
   <slots>
    <signal>XValueChanged(double)</signal>
    <signal>YValueChanged(double)</signal>
    <signal>ZValueChanged(double)</signal>
    <signal>PixelValueChanged(double)</signal>
    <slot>ShiftUp()</slot>
    <slot>SetOrientation(int)</slot>
    <slot>ShiftRight()</slot>
    <slot>ShiftLeft()</slot>
    <slot>ShiftDown()</slot>
    <slot>SetViewMode(int)</slot>
    <slot>Reset()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>IntensityMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IntensityMaxDisplay</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>623</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IntensityMaxDisplay</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IntensityMax</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>619</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IntensityMinDisplay</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IntensityMin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>584</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IntensityMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IntensityMinDisplay</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>588</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>Gui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>747</x>
     <y>658</y>
    </hint>
    <hint type="destinationlabel">
     <x>734</x>
     <y>667</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IntensityMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Window2D</receiver>
   <slot>IntensityMax(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IntensityMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Window2D</receiver>
   <slot>IntensityMin(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slice</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Window2D</receiver>
   <slot>ChangeSlice(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slice</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SliceNum</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliceNum</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slice</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>556</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ZoomIn</sender>
   <signal>clicked()</signal>
   <receiver>Window2D</receiver>
   <slot>ZoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ZoomOut</sender>
   <signal>clicked()</signal>
   <receiver>Window2D</receiver>
   <slot>ZoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Window2D</sender>
   <signal>XValueChanged(double)</signal>
   <receiver>PositionX</receiver>
   <slot>setNum(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Window2D</sender>
   <signal>YValueChanged(double)</signal>
   <receiver>PositionY</receiver>
   <slot>setNum(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Window2D</sender>
   <signal>ZValueChanged(double)</signal>
   <receiver>PositionZ</receiver>
   <slot>setNum(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Window2D</sender>
   <signal>PixelValueChanged(double)</signal>
   <receiver>PixelValue</receiver>
   <slot>setNum(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Up</sender>
   <signal>clicked()</signal>
   <receiver>Window2D</receiver>
   <slot>ShiftUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Orientation2D</sender>
   <signal>activated(int)</signal>
   <receiver>Window2D</receiver>
   <slot>SetOrientation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Right</sender>
   <signal>clicked()</signal>
   <receiver>Window2D</receiver>
   <slot>ShiftRight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Down</sender>
   <signal>clicked()</signal>
   <receiver>Window2D</receiver>
   <slot>ShiftDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Left</sender>
   <signal>clicked()</signal>
   <receiver>Window2D</receiver>
   <slot>ShiftLeft()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Display2D</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Window2D</receiver>
   <slot>SetViewMode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Center</sender>
   <signal>clicked()</signal>
   <receiver>Window2D</receiver>
   <slot>Reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
